### DISCUSSION TITLE
Android 14

### DISCUSSION THREAD

**Discussion by Vince56190** [fcaronte/KingInstaller#12] [Q&A]
Ne peut plus installer l'application AAtorque sur Android 14 sur téléphone pixel 6A , elle n'apparait plus dans Android Auto.
Can no longer install the AAtorque application on Android 14phone Pixel 6A , it no longer appears in Android Auto.

**Comment by fcaronte** [comment]
Do you have a Pixel phone? Probably it need the same trick used for the A13 version but using new A14 Google package. Need to try and let's say us

**Reply by Vince56190** [reply]
where i can find this trick ? you talk about galaxy A13 phone ?

**Reply by fcaronte** [reply]
> where i can find this trick ? you talk about galaxy A13 phone ? 
> 

Nono i talk only for pixel, I don't know if Samsung have new problems with last android version but I suggest to try reinstall the app check again unknown source on android auto developer mode

**Reply by Vince56190** [reply]
I did all that, it still doesn't work. This App worked with Android 14 but the update of the AAtorque application today is no longer done by Kinginstaller, it appears in my Apps but not in Android Auto

**Reply by jamesnorman93** [reply]
Good to know it's a clash with Kingstaller and Android 14 as it's happening on a Samsung too...
I thought it may have been just Pixel's...

AAAD installs "Performance Monitor" fine on my Android 14 Pixel 6.

**Comment by fcaronte** [comment]
Other android auto app like fermata auto are working with A14? Maybe simple
we need to way an update of aatorque app with A14 compatibility

Il sab 7 ott 2023, 17:02 Vince56190 ***@***.***> ha scritto:

**Reply by Vince56190** [reply]
Yes Fermata still working  with Android 13 but it was on my phone before updating to 14 like AATorque. AATorque worked during the Android 14 update but when updating AAtorque, it was Kinginstaller who no longer knew how to do it

**Reply by fcaronte** [reply]
Seem like previous installed app will stay, but if you remove and try to install again will not work so need to find a A14 trick too, if exist...

**Comment by pvolchek** [comment]
Hey, same issue here. Pixel 6 Pro. Trying to install 'aa-torque' for Android Auto (Torgue Pro)
Followed all the steps, yet the message is 'App installed from KingInstaller', and obviously not showing up on Android head unit.
Someone mentioned to reinstall the Goole's Package Installer, but honestly I have no idea how to do that. My apps do not even seem to have it (guess it is a system app).

**Reply by jamesnorman93** [reply]
Don't worry about trying. Reinstalling the Package 14 doesn't help Kingstaller on Android 14.

**Comment by fcaronte** [comment]
Is same trick you just done with android 13 on pixel, on previous version
need trick too

Il sab 7 ott 2023, 22:02 pvolchek ***@***.***> ha scritto:

**Reply by jamesnorman93** [reply]
I can confirm after attempting the package 14 install fix for the Pixel (6) that this fix no longer works with Android 14.

I did this same method on Android 13 last week (before I updated) and Kingstaller worked with the package installer 13 and aa-torque.

**Comment by fcaronte** [comment]
Seem that update dependency and sdk targets fix A16 problem but not on oppo\realme, so try and let's say if work for you  I test on my s24ultra and seem to work

**Comment by fcaronte** [comment]
Last update with shizuku mode seem to work for most phone, actually confirmed working on Honor, redmagic, Nothing and pixel

**Comment by fcaronte** [comment]
Last update with shizuku mode seem to work for most phone, actually confirmed working on Honor, redmagic, Nothing and pixel
